kratos/python/add_mesh_to_python.cpp

```diff
@@ -70,6 +60,11 @@ typedef MeshType::NodesContainerType NodesContainerType;
 typedef Geometry<Node<3> >::PointsArrayType NodesArrayType;
 typedef Geometry<Node<3> >::IntegrationPointsArrayType IntegrationPointsArrayType;
 
+array_1d<double,3> GetNormalFromCondition( Condition& dummy )
```

Comment:
this is exactly what i am telling, you are changing the API of the geometry class. This is forbidden unless an agreement is reached on that.

If you want to propose a change to the base-class, you must open an issue for that, and go through a discussion round. For example in this case you already have one feedback: returning a single normal per condition does NOT sound correct to me since it is not valid for high order geometries